all_seaing_utility/all_seaing_utility/rosbag_to_image.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+
+# python3 /home/arcturus/arcturus/dev_ws/src/all_seaing_vehicle/all_seaing_utility/all_seaing_utility/rosbag_to_image.py /home/arcturus/arcturus/test_05-21-24/test1/test1_0.db3 /zed/zed_node/rgb/image_rect_color /home/arcturus/arcturus/bag_output
+
+from argparse import ArgumentParser
+import cv2
+from cv_bridge import CvBridge
+import os
+import rosbag2_py
+from rclpy.serialization import deserialize_message
+from sensor_msgs.msg import Image
+import rclpy
+
+def main():
+    rclpy.init()
```

Comment:
I think for this script you won't need rclpy.init() or shutdown(). Maybe I'm wrong though, so you should check.

Reply:
Does anything bad happen if run init & shutdown when it's not needed?

Reply:
Nothing "bad" would happen, but I don't think it is necessary for this script? It's just nice to not add unused code.